README.rst

```diff
@@ -61,7 +61,7 @@ Optional Settings
 .. code-block:: python
 
 	GRAPH_AUTH = {
-	    'USER_FIELDS': ('email', 'first_name', 'last_name', ), # Which username fields are available
+	    'USER_FIELDS': ('username', 'email', 'first_name', 'last_name', ), # Which user fields are available
```

Comment:
@mariuszstr What happens if it's not included? In general I think we should default to including/requiring the minimum fields necessary for the functionality to work and let other users add more as desired.